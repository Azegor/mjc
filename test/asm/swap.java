class Foobar {
    public int a;
    public static void main(String[] args){
      int a = 10;
      int b = 20;

      int i = 0;
      while(i < 1) {
        int tmp = a;
        a = b;
        b = tmp;

        i = i + 1;
        System.out.println(1337);
      }

      System.out.println(a);
      System.out.println(b);
    }
}
