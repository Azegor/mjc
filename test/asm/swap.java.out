1337
20
10
